downstream/assemblies/platform/assembly-installing-automation-mesh.adoc

```diff
@@ -3,20 +3,24 @@ ifdef::context[:parent-context: {context}]
 
 
 [id="installing-automation-mesh"]
-= Installing {AutomationMesh}
+= Installing and configuring {AutomationMesh}
 
 
 :context: installing-automation-mesh
 
 
 [role="_abstract"]
-You can install an {AutomationMesh} using the instructions found in the {PlatformName} Installation Guide. {PlatformNameShort} installations are separated into unique scenarios based on the components deployed. You can incorporate {AutomationMesh} designs based on which platform components you intend to deploy in your environment. Review the systems requirements for each component and {AutomationMesh} node type before proceeding with your installation.
+You can install {AutomationMesh} using the instructions found in the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.1/html/red_hat_ansible_automation_platform_installation_guide/index[{PlatformName} Installation Guide]. {PlatformNameShort} installations are separated into unique scenarios based on the components deployed. You can incorporate {AutomationMesh} designs based on which platform components you intend to deploy in your environment.
+
+The following sections provide options for you to configure your {AutomationMesh}, including an option to import your own CA certificate.
+
+include::platform/proc-import-mesh-ca.adoc[leveloffset=+1]
```

Comment:
To be honest this assembly seems a little odd. It looks like it was created as a sort of "stub" assembly that didn't contain any include directives.

As a user the title doesn't quite match the abstract, which is basically a pointer to the installation guide. There's no procedure for installation in the user story so the high-level heading doesn't line up with the content.

It seems to me like we're kind of "bolting on" more content here and this assembly will turn into a hodge-podge sort of dump. Also, when I look at the title for the mesh docs it appears that this assembly is at the end, after deprovisioning.

What about moving this under the planning assembly before the example topologies? You could rename the assembly title to something like "Setting Up {AutomationMesh}". It would then make sense to me to break the first couple sentences into a new concept topic and expand out a little about installing mesh. Then the procedure about adding the CA cert.